ArjunaJTS/integration/src/test/java/com/arjuna/ats/jta/distributed/SimpleIsolatedServers.java

```diff
@@ -1065,7 +1065,8 @@ public void testMigrateTransactionSubordinateTimeout() throws Exception {
 			Thread.currentThread().setContextClassLoader(classLoader);
 		}
 		assertTrue("" + completionCounter.getRollbackCount("2000"), completionCounter.getRollbackCount("2000") == 1);
-		assertTrue("" + completionCounter.getRollbackCount("1000"), completionCounter.getRollbackCount("1000") == 2);
+		// The second rollback invocation (from the proxyXAResource) is ignored because of JBTM-XXXX, that's why the following condition is == 1
```

Comment:
@jmfinelli please can you add the full JBTM number here?

Reply:
Done. Thanks

Reply:
I think it's possible that this is still saying JBTM-XXXX (at least from viewing https://github.com/jbosstm/narayana/pull/2214/files#diff-f273d7bba38eaa0e38556825398072748c28164b6f6bb0a255792c70d7346745R1069)

Reply:
Duh

Reply:
Checking the code, it seems that JBTM-XXXX has been changed to JBTM-3843. Could you please let me know if you still see `JBTM-XXXX`? Thanks

Reply:
I don't think so  - please take a look at https://github.com/jmfinelli/narayana/blob/JBTM-3843/ArjunaJTS/integration/src/test/java/com/arjuna/ats/jta/distributed/SimpleIsolatedServers.java#L1069 for example?

Reply:
Double duh...the STYLE commit changed this back to JBTM-XXXX. Copy and paste mistake :face_with_head_bandage: I will redo the STYLE commit to be sure that I didn't make any other silly mistakes. Thanks for double checking Tom

Reply:
OK, now it seems that I managed to change the JBTM-XXXX :relieved: